src/main/java/seedu/address/model/util/SampleDataUtil.java

```
@@ -0,0 +1,55 @@
+                new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@gmail.com"),
+                    new Address("Blk 30 Geylang Street 29, #06-40"),
+                    new UniqueTagList("friends")),
+                new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@gmail.com"),
+                    new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
+                    new UniqueTagList("colleagues", "friends")),
+                new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@yahoo.com"),
+                    new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
+                    new UniqueTagList("neighbours")),
+                new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@google.com"),
+                    new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
+                    new UniqueTagList("family")),
+                new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@outlook.com"),
+                    new Address("Blk 47 Tampines Street 20, #17-35"),
+                    new UniqueTagList("classmates")),
+                new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@gmail.com"),
+                    new Address("Blk 45 Aljunied Street 85, #11-31"),
+                    new UniqueTagList("colleagues"))
+            };
+        } catch (IllegalValueException e) {
+            throw new AssertionError("sample data cannot be invalid", e);
+        }
+    }
+
+    public static ReadOnlyAddressBook getSampleAddressBook() {
+        AddressBook sampleAB = new AddressBook();
+        try {
+            for (Person samplePerson : getSamplePersons()) {
+                sampleAB.addPerson(samplePerson);
+            }
+        } catch (DuplicatePersonException e) {
+            assert false : "sample data cannot contain duplicate persons";
```

@damithc 
> Any reason why we are using assert here but AssertionError in the method above?

The usage above is so that we will not need to return a bogus value.

I personally would prefer `AssertionError` to be used here as well since we are handling an exception, but it's not required since we don't need to return any value here, although I guess it may be bad if we are swallowing exceptions if assertions are disabled. (But then again, assertions would only be disabled in production, and we would have caught this assertion error during testing, so it may not be a problem in practice. In summary, I don't feel strongly about this.)